python/tester/utils.py

```diff
@@ -117,7 +117,7 @@ def get_last_group(bucket, instrument, date):
         if numbers:
             last_number = max(numbers)
         else:
-            last_number = 0
+            last_number = 90000
```

Comment:
If this is a six-digit counter (per line 105), should this be 900_000 instead of 90_000?

Reply:
This is really `90_000`. 

One real life example of summit LSSTCam exposure ID=`2025072800142`, group=`2025-07-29T03:06:07.417`, obs_id=`MC_O_20250728_000142`. 

An example from our dev repo: exposure=`2025080600012`, group=`2025-08-06T00:00:00.000012`, obs_id=`MC_O_20250806_000012`. 

The 6 digits as per line 105 was just convenient as microseconds.  

And indeed `obs_id` has 6 digits there too.

Reply:
However I missed to see if I could use a different controller in the mocked exposure.  I'll look into that more.

Reply:
The Camera image name field has 6 digits, but the exposure ids in DM have only 5 as we didn't believe there could ever be more than 100K per day (more than one per second).  So you shouldn't use sequence numbers greater than 99999.